### DISCUSSION TITLE
Adguard base filter ada di mana?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1431] [Q&A] [answered]
Adguard base filter itu di bagian mana sih kak di adguard kok aku cari muter2 gk ketemu? Tolong screenshoot

**Comment by realodix** [comment]
Aplikasi desktop, android atau ekstensi browser?

**Reply by Jokopentil** [reply, marked as answer]
Saya habis lapor karena iklan youtube
Daya bingung nyari katanya aktifkan adguard base filter
Yg aku cari adguard versi terjemahan
Udah ketemu yg adalah adguard dasar filter
Ooo iya kak mau tanya setiap loading pertama kenapa adblock mati
Dan harus reload supaya iklan terblokir?

**Reply by realodix** [reply]
> Ooo iya kak mau tanya setiap loading pertama kenapa adblock mati
> Dan harus reload supaya iklan terblokir?

Kalau masalah itu harus tanya ke developernya kak, Saya juga kurang tahu kenapa. Btw di Saya pribadi tidak seperti itu
